docs/development/release_management.rst

```diff
@@ -22,59 +22,48 @@ Pre-Release
+
+#. Create a release branch.
+
+   For a regular release, create a release branch off of ``develop``::
 
      git checkout develop
-     git checkout -b release/1.x
+     git checkout -b release/<major>.<minor>.0
+
+
+   For a point release, create a release branch off of the latest merged release branch::
+
+     git checkout release/<major>.<minor>.0
+     git checkout -b release/<major>.<minor>.1
 
    .. warning:: For new branches, please ask a ``freedomofpress``
                 organization administrator to enable branch protection
                 on the release branch. We want to require CI to be
                 passing as well as at least one approving review prior
                 to merging into the release branch.
 
-#. For each release candidate, update the version and
-   changelog. Collect a list of the important changes in the release,
-   including their GitHub issues or PR numbers, then run the
-   ``update_version.sh`` script, passing it the new version in the
-   format ``major.minor.patch~rcN``, e.g.::
-
-     securedrop/bin/dev-shell ../update_version.sh 1.3.0~rc1
-
-   The script will open both the main repository changelog
-   (``changelog.md``) and the one used for Debian packaging in an
-   editor, giving you a chance to add the changes you collected. In
-   the Debian changelog, we typically just refer the reader to the
-   ``changelog.md`` file.
-
-#. If you would like to sign the release commit, you will need to do so manually:
-
-   a. Create a new signed commit and verify the signature::
-
-        git reset HEAD~1
-        git commit -aS
-        git log --show-signature
-
-   #. Ensure the new commit is signed, take note of the commit hash.
-
-   #. Edit ``1.x.y-rcN.tag`` and replace the commit hash with the new
-      (signed) commit hash.
+#. For each release candidate, update the version and changelog.
 
-   #. Delete the old tag and create a new one based on the tag file
-      edited above::
+   a. Collect a list of important changes in the release (see 
+      https://github.com/freedomofpress/securedrop/milestones) including GitHub issues or PR 
+      numbers for each change. You will add these changes to the changelog in the next step.
 
-         git tag -d 1.x.y-rcN
-         git mktag < 1.x.y-rcN.tag > .git/refs/tags/1.x.y-rcN
+   b. Run ``update_version.sh``, passing it the new version in the format 
+      ``<major>.<minor>.<patch>~rcN`` (note how we use a tilde here instead of a dash). The script 
+      will open both the main repository changelog (``changelog.md``) and the one used for Debian 
+      packaging in an editor, giving you a chance to add the changes you collected. In the Debian 
+      changelog, we typically just refer the reader to the ``changelog.md`` file:
+      ``securedrop/bin/dev-shell ../update_version.sh 1.3.0~rc1``
 
-#. Push the branch and tags:
+   c. Sign the commit added by the ``update_version.sh`` script:
+      ``git commit --all --amend --gpg-sign --no-edit``
```

Comment:
:rotating_light: Careful, that `--all` flag will take any local modifications in git repo and silently fold them into the previously created release commit. Remove the `--all` flag, then the command should do what we want: take the previously created commit, then sign it, without modifying the contents of the diff.

Reply:
removed `--noedit` as well just in case